src/misc/lv_event.c

```diff
@@ -109,12 +127,12 @@ bool lv_event_remove_dsc(lv_event_list_t * list, lv_event_dsc_t * dsc)
     LV_ASSERT_NULL(list);
     LV_ASSERT_NULL(dsc);
 
-    int size = lv_array_size(list);
-    lv_event_dsc_t ** events = lv_array_front(list);
+    int size = lv_array_size(EVENT_ARRAY(list));
```

Comment:
uint32_t size

Reply:
I didn't modify this, and I don't want to introduce changes unrelated to the current patch in the current patch.